### PR TITLE
Bugfix for cliff-lorimer quantification

### PR DESCRIPTION
**Pull request by thomasaarholt**
### Description of the change
- There would be an an error on quantifying a single array of intensities due to a typo
- Several `if absorption_correction is not None` led to `absorption_correction = absorption_correction`, which was unnecessary.
I'm afraid that I don't have time to add tests.
 
### Progress of the PR
- [x] Change implemented (can be split into several points),
- [x] ready for review.

### CODE REVIEW

**Comment by ericpre** [comment]
Thanks @thomasaarholt, but as you would have anticipated, this would a test! If you don't have time, then someone else should finish the PR.
From the coverage, it seems that the missing case is: CL quantification of a single spectrum with `absorption_correction=None`.

**Comment by jlaehne** [comment]
Does anyone have time to add tests? Should be low hanging fruits.

File: hyperspy/_signals/eds_tem.py [hunk 1]
```diff
@@ -559,7 +559,7 @@ def quantification(self,
             utils.plot.plot_signals(composition, **kwargs)
 
         if absorption_correction:
-            _logger.info(f'Conversion found after {it} interations.')
+            _logger.info(f'Convergence reach after {it} interations.')
```
Comment (jlaehne) [inline hyperspy/_signals/eds_tem.py]:
```suggestion
            _logger.info(f'Convergence reached after {it} interations.')
```

File: hyperspy/_signals/eds_tem.py [hunk 2]
```diff
@@ -399,13 +399,22 @@ def quantification(self,
         --------
         vacuum_mask
         """
+        if (not isinstance(intensities, (list, tuple)) or
+                not isinstance(intensities[0], BaseSignal)):
+            raise ValueError(
+                "The parameter `intensities` must be a list of signal."
```
Comment (jlaehne) [inline hyperspy/_signals/eds_tem.py]:
```suggestion
                "The parameter `intensities` must be a list of signals."
```

**Review by jlaehne — APPROVED** [review]

**Comment by jlaehne** [comment]
add changelog entry?

**Review by jlaehne — APPROVED** [review]

**Pull request merged**